src/components/Breadcrumb/index.js

```diff
@@ -0,0 +1,137 @@
+import React, { Fragment } from 'react'
+import { PAGE_TYPE } from 'constants'
+import { useGlobalContext } from 'providers/GlobalProvider'
+import Box from '@material-ui/core/Box'
+import {
+  AiOutlineBranches,
+  AiOutlineFile,
+  AiOutlineAppstore,
+  AiOutlineGithub,
+  AiOutlinePullRequest,
+  AiOutlineFieldNumber,
+  AiOutlineRight,
+} from 'react-icons/ai'
+
+const Breadcrumb = ({
+  pageType,
+  owner,
+  repo,
+  commit,
+  pull,
+  branch,
+  filePath,
+}) => {
+  const { togglePullMenu, togglePullCommitMenu } = useGlobalContext()
+
+  const items = [
+    { text: owner, icon: AiOutlineGithub },
+    { text: repo, icon: AiOutlineAppstore },
+  ]
+
+  const branchItem = { text: branch, icon: AiOutlineBranches }
+  const fileItem = { text: filePath, icon: AiOutlineFile }
+  const commitItem = {
+    text: commit,
```

Comment:
Maybe we can slice the commit value, it's way too long
![image](https://user-images.githubusercontent.com/20126676/139564183-fe21316a-0239-4c22-a67f-2ac8051bae95.png)

Reply:
fixed, thanks